### DISCUSSION TITLE
Can the code be exported/saved?

### DISCUSSION THREAD

**Discussion by Rc85** [LiveDuo/destack#120] [Q&A]
I'm building an ecommerce website for customers to list their products. In the product description, I want them to be able to create and edit it so it looks like a website and feels interactive. An example would be [this amazon's product page](https://www.amazon.ca/Tineco-iFLOOR-Breeze-Complete-Cordless/dp/B0BVZHB2KY/?_encoding=UTF8&pd_rd_w=o8OmW&content-id=amzn1.sym.6ab822bb-4b3a-4cfa-8197-4ab3cee1b8d9&pf_rd_p=6ab822bb-4b3a-4cfa-8197-4ab3cee1b8d9&pf_rd_r=4SZA1Z9Q0AZDS1X1XA53&pd_rd_wg=wcUEW&pd_rd_r=7842fe4e-56d0-4648-8fff-c5607729b593&ref_=pd_hp_d_atf_unk&th=1) where the description have sliders, buttons, etc. Can destack achieve this?

**Comment by LiveDuo** [comment]
Hi @Rc85,

Anything you can do with pure HTML/JS can be done in Destack. In this case, you would need to add a script to control the buttons or sliders. You can add this script either in DevTools or by editing the HTML template file.

How to use DevTools with Destack:

https://private-user-images.githubusercontent.com/370612/318197307-430540a1-b65d-4be6-8091-fae88d14a41a.mov
